src/ng-generate/components/form/generators/components/fields/datePartial/files/__name@dasherize__.component.html.template

```diff
@@ -4,6 +4,7 @@
         type="text"
         matInput
         placeholder="<%= options.fieldConfig.placeholder %>"
+        data-test="<%= options.fieldConfig.placeholder %>"
```

Comment:
Should we use `options.fieldConfig.name` instead of a placeholder? I'm not sure that placeholder will always have a string that fits well for being used as an ID.

Reply:
done